src/Shared/Recruit.Vacancies.Client/Infrastructure/Repositories/MongoDbVacancyReviewRepository.cs

```diff
@@ -79,19 +83,21 @@ public async Task<List<VacancyReview>> GetByStatusAsync(ReviewStatus status)
             return result;
         }
 
-        public async Task<List<VacancyReview>> GetActiveAsync()
+        public async Task<List<T>> GetActiveAsync<T>()
         {
-            var filterBuilder = Builders<VacancyReview>.Filter;
+            var filterBuilder = Builders<T>.Filter;
 
-            var filter = filterBuilder.Eq(r => r.Status, ReviewStatus.PendingReview)
-                         | filterBuilder.Eq(r => r.Status, ReviewStatus.UnderReview);
+            var filter = filterBuilder.Eq(Status, ReviewStatus.PendingReview.ToString())
```

Comment:
Could use a base class and put a generic filter `where T: BaseClass` and carry on using Lambda expression here.

Reply:
At the moment it works with any type you throw at it. I don't think making it tighter coupled is worth the lambda expression payoff.

Reply:
I think it is a matter of setting expectations rather than tight coupling.

Reply:
There is no reason why the Type you are projecting to should contain the properties your are filtering on. In this case were projecting to `VacancyReviewSummary` which doesn't have `Status` as a property.